src/components/notebook-utils.js

```diff
@@ -16,8 +16,8 @@ import validate from 'validate.js'
 export const notebookNameValidator = existing => ({
   presence: { allowEmpty: false },
   format: {
-    pattern: /^[^#[\]*?]*$/,
-    message: 'can\'t contain any of these characters: "#[]*?"'
+    pattern: /^[^#[\]*?:;@$%+=\\,/]*$/,
+    message: 'can\'t contain any of these characters: # [ ] * ? , : ; @ $ % + = / \\ '
```

Comment:
Does anyone know how to add a newline to the message? `\n` and `\r` did not work.